src/flint/__init__.py

```diff
@@ -20,6 +20,9 @@
 from .types.fmpz_mod_poly import *
 from .types.fmpz_mod_mat import fmpz_mod_mat
 
+from .types.fmpq_mpoly import *
+from .types.fmpz_mpoly_q import *
```

Comment:
I would prefer to avoid start imports here. It would be better to list what is imported explicitly.

I realise that most other modules all have star imports and there is no need to fix that in this PR.